src/main/java/com/hubspot/imap/utils/parsers/string/BufferedBodyParser.java

```diff
@@ -79,10 +82,10 @@ public BufferedBodyParser(SoftReferencedAppendableCharSequence sequenceRef) {
             return Optional.empty();
           }
 
-          String result = buf.toString(StandardCharsets.UTF_8);
```

Comment:
are we sure that the callers of this method do this conversion themselves?

Reply:
The ResponseDecoder is the only caller of this method and all it does is take the String that we created here and then get the bytes back out of it with `body.get().getBytes(StandardCharsets.UTF_8)`